test/src/Test.files

```diff
@@ -6,3 +6,4 @@ T_SRCS := \
   state_machine/dummy.test.cpp \
   utils/config.test.cpp \
   utils/differentiator.test.cpp \
+  utils/kalman_multivariate.test.cpp \
```

Comment:
You are going to have a merge conflict with this file, but it should be easy to solve.